packages/rest/src/lib/RequestManager.ts

```diff
@@ -181,6 +191,41 @@ export class RequestManager extends EventEmitter {
 		this.options = { ...DefaultRestOptions, ...options };
 		this.options.offset = Math.max(0, this.options.offset);
 		this.globalRemaining = this.options.globalRequestsPerSecond;
+
+		setInterval(() => {
```

Comment:
If someone sets the hash sweep interval to infinity or 0 you should not set this interval